lib/Slic3r/GUI/Plater/SplineControl.pm

```diff
@@ -267,7 +267,11 @@ sub _update_canvas_size {
     my @size = ($canvas_w - 2*$padding, $canvas_h - 2*$padding);
     $self->{canvas_size} = [@size];
     
-    $self->{scaling_factor_x} = $size[0]/($self->{max_layer_height} - $self->{min_layer_height});
+    if ($self->{max_layer_height} == $self->{min_layer_height}) {
```

Comment:
Yes. But a max_layer_height small than min_layer_height doesn't make any sense anyway. So I suggest:
```suggestion
    if ($self->{max_layer_height} > $self->{min_layer_height}) {
        $self->{scaling_factor_x} = $size[0]/($self->{max_layer_height} - $self->{min_layer_height});
    } else {
        $self->{scaling_factor_x} = 1.0;
    }
```
Additionally, we should catch this case when sanitizing the config and request the user to enter useful values.

Reply:
Done.